CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.31
```

Comment:
This should be 0.12.30+1, since there aren't any new features being added.

Reply:
Done.